eng/pipelines/common/variables.yml

```diff
@@ -8,9 +8,9 @@ variables:
 - name: DOTNET_VERSION
   value: 10.0.100-preview.2.25164.34
 - name: REQUIRED_XCODE
-  value: 16.2.0
+  value: 16.3.0
```

Comment:
Moving to Xcode 16.3.0 . This only works if start the build from AZDO and enable provisionator on the public ci. On devdiv It works